docs/src/contribute/governance.md

```diff
@@ -32,12 +32,17 @@ Website Team Members are community members who have shown that they are committe
 
  Website Team Members:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on Discord once per week day for team updates.
```

Comment:
I think mentioning the `#team` Discord channel here would be a good idea.

Reply:
I wasn't sure if I should do that because `#team` isn't public, but this document is.... :smile:

But since it's mentioned in this comment, I suppose not a big deal to add it.